rebar.config.script

```diff
@@ -47,7 +47,20 @@ begin
 
   Data = ["-define(_internal_otp_vsn_", X,$_,Y,$_,Z, ", true).", $\n],
 
-  DefaultBuildLibDir = "_build/default/lib",
+  FLog = fun(Fmt,Args) ->
+          case os:getenv("DEBUG") of
+            "1" ->         %% env var DEBUG set to 1
+              io:format(Fmt,Args);
+            _ ->           %% env var DEBUG not defined/empty string/0...
+              ok
+            end
+          end,
+
+  %% DefaultBuildLibDir = "_build/default/lib",
+  DefaultBaseDir = proplists:get_value(base_dir,CONFIG,"_build"),
+  DefaultBuildLibDir = DefaultBaseDir ++ "/default/lib",
+
+  FLog("DefaultBuildLibDir=~p~n",[DefaultBuildLibDir]),
```

Comment:
Looks like this will crash without the right env.

Reply:
I test with DEBUG=1/2 and without DEBUG defined in env, all pass rebar3 eunit test.